event-system/docs/API.md

```diff
@@ -0,0 +1,199 @@
+Event system overview
+===
+
+Microservices can publish events to clients by using `POST /publish/:namespace/:room`. The
+namespace shoud uniquely identify the microservice. It is used to determine where to send
+acl requests to when determining if a user is authorized to subscribe to an event stream.
+
+The event system writes the event to the redis datastore. All connected event system
+instances are subscribed to the redis notification stream. When redis notifies an
+event system about a new event, it writes the new event to all subscribed clients.
+
+Clients are required to send an `Authorize` header for identification with every request.
+
```

Comment:
On line 6 of authentication.md you are calling it 'authorization header'. Is it `Authorize` or `Authorization`?